presto-native-execution/presto_cpp/main/types/PrestoToVeloxExpr.cpp

```diff
@@ -67,7 +67,19 @@ std::string mapScalarFunction(const std::string& name) {
 }
 
 std::string mapAggregateOrWindowFunction(const std::string& name) {
-  return boost::to_lower_copy(name);
+  static const std::unordered_map<std::string, std::string>
+      kAggregateFunctionsMap = {
```

Comment:
kAggregateFunctionsMap -> kFunctionNames for readability; 'Aggregate' is already part of the encoding function name.